README.md

```diff
@@ -3,7 +3,7 @@
 </p>
 
 <p align="center">
-  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/workflow/status/laravel-zero/framework/Tests.svg" alt="Build Status"></img></a>
+  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml" alt="Build Status"></img></a>
```

Comment:
Could you change this to use the `master` branch for the status? Other than that, LGTM.
```suggestion
  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"></img></a>
```